.travis.yml

```diff
@@ -20,6 +20,9 @@ matrix:
     - env:
       - DIST=ubuntu
       - VERSION=14.04
+    - env:
+      - DIST=ubuntu
+      - VERSION=16.04
```

Comment:
Could you please update `before_install` section to install DEB using http://download.jubat.us/apt/ubuntu/xenial so that this config work on master branch?